webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Parameters/ParametersTable.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { TableComposable, Thead, Tbody, Tr, Th } from '@patternfly/react-table';
+import {
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+  Button,
+  Spinner,
+  SearchInput,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../../../common/I18n';
+import { ParametersTableRow } from './TableRow';
+import { columnNames, HOST_PARAM } from './ParametersConstants';
+import Pagination from '../../../Pagination';
+import { useForemanSettings } from '../../../../Root/Context/ForemanContext';
+import './Parameters.scss';
+import { STATUS } from '../../../../constants';
+
+export const ParametersTable = ({
+  status,
+  hostId,
+  allParameters,
+  editHostsPermission,
+}) => {
+  const [editingRow, setEditingRow] = useState(-1);
+  const [activeSortDirection, setActiveSortDirection] = useState('asc');
+  const { perPage: settingsPerPage = 20 } = useForemanSettings() || {};
+  const [page, setPage] = useState(1);
+  const [perPage, setPerPage] = useState(settingsPerPage);
+  const [showNewRow, setShowNewRow] = useState(false);
+  const [search, setSearch] = useState('');
+
+  const sortedParams = (
+    direction = activeSortDirection,
+    params = allParameters
+  ) =>
+    [...params].sort((a, b) => {
+      if (direction === 'asc') {
+        return a.name.localeCompare(b.name);
+      }
+      return b.name.localeCompare(a.name);
+    });
+  const [parameters, setParameters] = useState(
+    sortedParams().slice(0, settingsPerPage)
+  );
+  const sortParams = {
+    sortBy: {
+      index: 0,
+      direction: activeSortDirection,
+      defaultDirection: activeSortDirection,
+    },
+    onSort: (_event, index, direction) => {
+      setActiveSortDirection(direction);
+      setParameters(sortedParams(direction).slice(0, perPage));
+    },
+    columnIndex: 0,
+  };
+  const onPaginationChange = ({ page: newPage, per_page: newPerPage }) => {
+    setPage(newPage);
+    setPerPage(newPerPage);
+    setParameters(
+      sortedParams().slice((newPage - 1) * newPerPage, newPage * newPerPage)
+    );
+  };
+  const onSearch = newSearch => {
+    setSearch(newSearch);
+    setParameters(
+      sortedParams(
+        activeSortDirection,
+        allParameters.filter(param => param.name.includes(newSearch))
+      ).slice(0, settingsPerPage)
+    );
+  };
+  return (
+    <>
+      <Toolbar>
+        <ToolbarContent>
+          <ToolbarItem variant="search-filter">
+            <SearchInput
+              placeholder={__('Find by name')}
+              value={search}
+              onChange={setSearch}
+              onSearch={onSearch}
+              onClear={() => setSearch('')}
+            />
```

Comment:
Can we eliminate the right arrow search button?

![image](https://user-images.githubusercontent.com/22042343/201760619-804e94c2-c368-48fb-bad7-335f84724c9a.png)


For reference, here's what it looks like on the Errata tab:
![image](https://user-images.githubusercontent.com/22042343/201760715-2fa32427-6638-4334-8912-f196dde9d9cb.png)

and on Reports:
![image](https://user-images.githubusercontent.com/22042343/201760760-863caed0-1387-4fcf-90f5-ca3efd9b6633.png)

This would be the third different look for the search bar, all on the same host details page. I know there's a lot of technical limitations that prevent perfect consistency here, but let's see if we can do anything.

Reply:
I dont mean to make it your problem, but since the PF search has an error and no easy way of changing it, can we all move to use the error?
It will also be the default in foreman searches after this pr: https://github.com/theforeman/foreman/pull/9277

Reply:
Decided in a meeting to move to an arrow icon in all searches